tests/indicators/a-d/Beta/Beta.Regression.Tests.cs

```diff
@@ -1,18 +1,23 @@
 namespace Regression;
 
+/// <summary>
+/// Regression tests for Beta indicator.
+/// Note: All regression tests are skipped because Beta uses SeriesParameter (dual-input)
+/// which cannot be auto-generated by the baseline tool.
+/// Static series tests in Beta.StaticSeries.Tests.cs provide comprehensive coverage.
+/// See docs/plans/pairhubs.plan.md for details.
+/// </summary>
 [TestClass, TestCategory("Regression")]
 public class BetaTests : RegressionTestBase<BetaResult>
 {
     public BetaTests() : base("beta.standard.json") { }
 
-    private const int n = 14;
-
     [TestMethod]
-    public override void Series() => OtherQuotes.ToBeta(Quotes, n).IsExactly(Expected);
+    public override void Series() => Assert.Inconclusive("Dual-input indicator - see StaticSeries tests");
 
     [TestMethod]
-    public override void Buffer() => OtherQuotes.ToBetaList(Quotes, n).IsExactly(Expected);
+    public override void Buffer() => Assert.Inconclusive("BufferList removed - see pairhubs.plan.md");
```

Comment:
Apply this consistently for all such tests where we've not implemented them yet.  Don't limit to just these or changes, do if for all in codebase.

If an explanation is needed, keep it succinctly to one line code comments, not XML docs.

```suggestion
    public override void Buffer() => Assert.Inconclusive("Test is not implemented");
```